src/libaktualizr/utilities/events.h

```diff
@@ -117,6 +117,18 @@ class DownloadComplete : public BaseEvent {
   static DownloadComplete fromJson(const std::string& json_str);
 };
 
+/**
+ * An target has been successfully downloaded.
```

Comment:
Minor: "*A* target has been..."

But while I'm thinking about it, it would be nice if the event names for download and install were similar for one vs all. I like the sound of `DownloadTargetComplete`, but maybe `DownloadComplete` should be `AllDownloadsComplete` to match the installation. (Similarly, `InstallComplete` should perhaps be renamed `InstallTargetComplete`.)